report_pre_thesis/src/index.tex

```diff
@@ -73,7 +73,9 @@ \section{Related work}
 % \subsection{comment-end}
 
 \section{Definition}
-Given a set $E$ of elements detectable using LIBS, and considering \(\mathbb{P}(E)\) as the power set of $E$ which represents all possible combinations of these elements, we define \(K \subseteq \mathbb{P}(E) \) as the set of compositions of these elements that can be subjected to LIBS. Additionally, let $S$ be the set of all possible spectra emitted from these compositions.
+Given a set $E$ of elements detectable using LIBS, and considering \(\mathbb{P}(E)\) as the power set of $E$ which represents all possible combinations of these elements, we define \(K \subseteq \mathbb{P}(E) \) as the set of compositions of these elements that can be subjected to LIBS.
```

Comment:
I think this can be made clearer.

$E$ has, as of our current understanding, at most 118 elements (how many there are in the periodic table). Then $\mathbb{P}(E)$ would be of size $2^{118}=3.32307e+35$.

And then we limit our consideration to $K$, a subset of $\mathbb{P}(E)$, to some number that I personally don't know - I suspect none of us do.

In reality, the interest of the ChemCam MOC model is only predicting the compositions of 8 major oxides.
Could we simply give those and state that we intend to only look at those?

In other words:
Does the implication of us including (unknown, to us) other elements in our definitions benefit our analysis?
I don't know whether it could, but I think we may benefit from simply sticking to what the experts do (in the papers we use as baseline).

Also, just as an aside: isn't it convention to denote the power-set by $\mathcal{P}(E)$? In my experience, $\mathbb{P}$ is primarily used to denote number fields, e.g. $\mathbb{R}$.